Add built-in graph states to flamingpy.utils (#68)
XanaduAI/flamingpy: .github/CHANGELOG.md

## Release 0.8.2a5 (development release)

### New features since the last release
* Add new functions to create different graph states (GHZ, Ring, Linear, Star, Bell).

### Bug fixes

> * Small fix in `viz.draw_EGraph` that raised an error whenever a graph state with non-integer coordinates was plotted.

Reviewer: ```suggestion
* Small fix in `viz.draw_EGraph` that raised an error whenever a graph state with non-integer coordinates was plotted. [#68](https://github.com/XanaduAI/flamingpy/pull/68)
```

### Improvements

### Documentation changes

> * Mentions the new graph state functions from `flamingpy.utils.graph_states` in the `run_graph_states.py` tutorial. 

Reviewer: ```suggestion
* Mention the new graph state functions from `flamingpy.utils.graph_states` in the `run_graph_states.py` tutorial. [#68](https://github.com/XanaduAI/flamingpy/pull/68)
```

## Release 0.8.2a5 (development release)

### New features since the last release

> * Add functions to create different graph states (star and complete graphs, ring graphs, linear clusters, and Bell pairs) in a new module, `utils.graph_states`. [#68](https://github.com/XanaduAI/flamingpy/pull/68). (backward compatible)

Reviewer: ```suggestion
* Add functions to create different graph states (star and complete graphs, ring graphs, linear clusters, and Bell pairs) in a new module, `utils.graph_states`. [#68](https://github.com/XanaduAI/flamingpy/pull/68) (backward compatible)
```
